File: wlxx-services/wlxx-system/wlxx-system-client/src/main/java/com/xzg/wlxx/system/client/entity/po/TokenPo.java
package com.xzg.wlxx.system.client.entity.po;

import com.baomidou.mybatisplus.annotation.TableName;
import com.xzg.wlxx.common.base.BasePo;
import lombok.*;

/**
 * @author XiaoZG
 */
@TableName("token")
@Data
@EqualsAndHashCode(callSuper = true)
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class TokenPo extends BasePo<TokenPo> {

    private String token;
    private Boolean expired;
    private Boolean revoked;
    private Long userId;
}

File: wlxx-services/wlxx-system/wlxx-system-client/src/main/kotlin/com/xzg/wlxx/system/client/entity/po/TokenPo.kt
package com.xzg.wlxx.system.client.entity.po

import com.baomidou.mybatisplus.annotation.TableName
import com.xzg.wlxx.common.base.BasePo
import lombok.*

/**
 * @author XiaoZG
 */
@TableName("token")
@Data
@EqualsAndHashCode(callSuper = true)
@NoArgsConstructor
@AllArgsConstructor
@Builder
class TokenPo(
    var token: String? = null,
    var expired: Boolean? = null,
    var revoked: Boolean? = null,
    var userId: Long? = null
) : BasePo<TokenPo?>()
